### DISCUSSION TITLE
Change the location in one drive

### DISCUSSION THREAD

**Discussion by AFPhome** [lavinir/hassio-onedrive-backup#192] [Ideas]
Hello

Is there a possibility to store the backup into an other map than "apps" ?
With other words I would like to store the backup in a map that have created myself in one drive
Thx in advance

**Comment by lavinir** [comment]
This is explained in the [Readme](https://github.com/lavinir/hassio-onedrive-backup?tab=readme-ov-file#backup-location-in-onedrive)
Also : #77

**Comment by AFPhome** [comment]
Thx for your answer 👍
